test/e2e/specs/block-icons.test.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * Internal dependencies
+ */
+import '../support/bootstrap';
+import {
+	newPost,
+	newDesktopBrowserPage,
+	insertBlock,
+	searchForBlock,
+} from '../support/utils';
+import { activatePlugin, deactivatePlugin } from '../support/plugins';
+
+const INSERTER_BUTTON_SELECTOR = '.components-popover__content .editor-block-types-list__item';
+const INSERTER_ICON_SELECTOR = `${ INSERTER_BUTTON_SELECTOR } .editor-block-types-list__item-icon`;
+const INSPECTOR_ICON_SELECTOR = '.edit-post-sidebar .editor-block-icon__colors';
+
+async function getInnerHTML( selector ) {
+	return await page.$eval( selector, ( element ) => element.innerHTML );
+}
+
+async function getBackgroundColor( selector ) {
+	return await page.$eval( selector, ( element ) => {
+		return window.getComputedStyle( element ).backgroundColor;
+	} );
+}
+
+async function getColor( selector ) {
+	return await page.$eval( selector, ( element ) => {
+		return window.getComputedStyle( element ).color;
+	} );
+}
+
+async function getFirstInserterIcon() {
+	return await getInnerHTML( INSERTER_ICON_SELECTOR );
+}
+
+describe( 'Correctly Renders Block Icons on Inserter and Inspector', () => {
```

Comment:
Why not use Block Icons here to make snapshots smaller? There is some duplication with the name of individual tests.

Reply:
I removed the snapshots. We now use another validatiom mechanism so we don't have duplication of snapshots.